### DISCUSSION TITLE
Not downloading Images & Videos

### DISCUSSION THREAD

**Discussion by everestokok** [TumblThreeApp/TumblThree#360] [Q&A (Question & Answer)]
Today I tried to download a tumblr (https://dragon-discourse.tumblr.com/).

It downloaded blogs text, img_urls, video_urls.

But It is not downloading images, videos.

Why?

**Comment by thomas694** [comment]
Let's see.
First the blog is working. But as you say it's only saving image and video urls, you have to change the blog's settings, click on the blog in the left list, then the details tab on the right side. You have to enable the options "download images" and "download videos" and disable "download url list".

**Comment by everestokok** [comment]
Looks like I messed up with settings.

I already enabled 'Download Images' & 'Download Videos'.

But I also enabled 'Download URL list', never read the preview tips.

That is what confused me.

Now it is downloading Images.
Thanks man  :)

Little suggestion, make this "Download URL list" to something like "Skip download & Save URLs".
So that it is more reasonable :)
